apps/emqx/src/emqx_router.erl

```diff
@@ -73,6 +73,7 @@
 %%--------------------------------------------------------------------
 
 mnesia(boot) ->
+    mria_config:set_dirty_shard(?ROUTE_SHARD, true),
```

Comment:
Q: how can we know if it's safe to set a shard as dirty?
I remember during our tests that this shard had several transaction (non-dirty TIDs), but I may be remembering wrong. :thinking:

Reply:
There are no `mria:ro_transaction` calls for the shard. Regular transactions don't prevent using dirty import, since they run on the core nodes.